tests/tests_integration/test_api/test_templates.py

```diff
@@ -75,9 +77,15 @@ def new_template_instance(cognite_client, new_template_group_version):
 
 @pytest.fixture
 def new_view(cognite_client, new_template_group_version):
-    events = []
-    for i in range(0, 1001):
-        events.append(Event(external_id="test_evt_templates_1_" + str(i), type="test_templates_1", start_time=i * 1000))
+    len_new_view = 1000
+    events = [
+        Event(
+            external_id=f"test_evt_templates_1_{i}",
+            type="test_templates_1",
+            start_time=i * len_new_view,
+        )
+        for i in range(len_new_view + 1)
+    ]
```

Comment:
```suggestion
    events = [
        Event(external_id=f"test_evt_templates_1_{i}", type="test_templates_1", start_time=i * len_new_view)
        for i in range(1001)
    ]
```

Reply:
This won't work because now `len_new_view` is undefined.

Reply:
Just use a literal, that was my point 😄

Reply:
Yeah but with mine you only have one place to update it in case it needs to be changed.

Reply:
But they are not the same variable! First says do 1000 loops, the other says change start time to be seconds, not ms

Reply:
Ah whoops! Misread it then. Good thing I already made the change though.